bootstrapvz/plugins/admin_user/tasks.py

```diff
@@ -69,7 +94,7 @@ def run(cls, info):
 			                'dpkg-query', '-W', 'openssh-server'])
 			from bootstrapvz.common.tools import sed_i
 			sshdconfig_path = os.path.join(info.root, 'etc/ssh/sshd_config')
-			sed_i(sshdconfig_path, 'PermitRootLogin yes', 'PermitRootLogin no')
+			sed_i(sshdconfig_path, 'PermitRootLogin (yes|without-password|forced-commands-only)', 'PermitRootLogin no')
```

Comment:
Why would it be anything else than `PermitRootLogin yes`? It's a fresh Debian install we are modifying, there's no need to account for other configurations.